app/processing/glos/find-glos-control-file.js

```diff
@@ -0,0 +1,14 @@
+const { retry } = require('../../retry')
+const { getFile } = require('../../storage')
+
+const findGlosControlFile = async (paymentFile) => {
+  try {
+    await retry(() => getFile(paymentFile.replace(/(.dat)$/g, '.ctl')))
+  } catch (err) {
+    console.log(`unable to find control file for ${paymentFile}`)
```

Comment:
what should happen to the files when a .ctl cant be found here?
Quarantine payment file?
Wait indefinetly? So fc payment files will remain in the inbound folder?

Reply:
I'd suggest if the retry exceeds it's count then throw an error for that file and move on to the next file.

Some important considerations though:
- the sequence number for glos must not have been updated otherwise it may process the next file before the current one if the next file has it's control file, also means that this file would fail the sequence validation check on retry
- the process will just move on to the next file in the array without issue, eg if an SFI payment file is also waiting, this throwing an error can't block that being processed.
- next polling interval will just try this process again fresh